x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/core/security/transport/netty4/SecurityNetty4Transport.java

```diff
@@ -149,6 +149,19 @@ protected ChannelHandler getClientChannelInitializer(DiscoveryNode node, Connect
         return new SecurityClientChannelInitializer(node, connectionProfile);
     }
 
+    @Override
+    protected InboundPipeline getInboundPipeline(boolean isRemoteClusterChannel) {
```

Comment:
Sorry for another nit: Can we rename `isRemoteClusterChannel` here and in a few other places to `isRemoteClusterServerChannel`? The difference between server and client channels is rather important since the new logic is for server channel only.

Reply:
Pushed [3604757](https://github.com/elastic/elasticsearch/pull/98692/commits/360475700a827b015a23d44f2611ae68d621b626)